src/software/ai/hl/stp/play/ball_placement_play_test.py

```diff
@@ -11,10 +13,13 @@
 from proto.ssl_gc_common_pb2 import Team
 from proto.ssl_gc_geometry_pb2 import Vector2
 
+# test duration global constant
+TEST_DURATION = 20
+
 
 @pytest.mark.parametrize(
-    "run_enemy_ai", [False]
-)  # , True])  TODO: Issue 2780 re-enable this test
+    "run_enemy_ai", [(False,), (True,)]
+)  # TODO (#2690): Robot gets stuck in corner of defense area
```

Comment:
```suggestion
    "run_enemy_ai", [False, True]
)  # TODO (#2690): Robot gets stuck in corner of defense area
```
The brackets around the False makes the `run_enemy_ai` variable a tuple, instead of a boolean. So the if statement lower down doesn't actually behave as expected and the enemy team still runs AI.

Reply:
I updated both instances of the situation. The other one for the next test for free kick.

Reply:
Did it end up working as expected for you as well?

Reply:
yes